akka-actor/src/main/java/akka/japi/pf/ReceiveBuilder.java

```diff
@@ -270,13 +294,28 @@ public boolean defined(Object o) {
    * @return a builder with the case statement added
    */
   public ReceiveBuilder matchAny(final FI.UnitApply<Object> apply) {
-    addStatement(new UnitCaseStatement<Object, Object>(new FI.Predicate() {
-      @Override
-      public boolean defined(Object o) {
-        return true;
-      }
-    }, apply));
+    addStatement(new UnitCaseStatement<>(ALWAYS_TRUE,
+                                         apply));
+    return this;
+  }
+
+  /**
+   * Add a new case statement to this builder, that matches any argument.
```

Comment:
Any argument that matches the predicate, at least?

Reply:
True，will adjust it.

Reply:
@raboof I have updated it, thanks.